### DISCUSSION TITLE
What is the best way to communicate with others?

### DISCUSSION THREAD

**Discussion by grd** [helloSystem/hello#338] [General]
I am using helloSystem. I want to communicate with others because I have a couple of bugs that I want to discuss and have some ideas. What is the best way to do that? Should I use this Discussions group, file bugs, use Matrix or is there something else?

**Comment by grd** [comment]
No-one? Is this the only way to communicate with others?

**Reply by probonopd** [reply]
https://hellosystem.github.io/docs/developer/contact

**Comment by probonopd** [comment]
Welcome to helloSystem.

> Should I use this Discussions group, file bugs, use Matrix or is there something else?

Any of those.

**Comment by grd** [comment]
Thanks. I am gonna use Matrix.
